airmozilla/main/templates/main/event.html

```diff
@@ -334,6 +335,24 @@ <h1 class="entry-title">
         </div>
       {% endif %}
 
+      {% if event.is_public() %}
+        <div class="tab">
+          <input type="radio" id="tab-share" name="tab-group-1">
+          <label for="tab-share">Share</label>
+          <div class="content">
+            <div>
```

Comment:
that's probably better. The day we have a 4th or 5th or Nth link, we'll have to consider changing to...
```
<p>
    <a ...>...</a><br>
    <a ...>...</a><br>
    <a ...>...</a><br>

    <a ...>...</a><br>
    <a ...>...</a><br>
    <a ...>...</a><br>
</p>
```